context/app/static/js/components/organ/Azimuth/Azimuth.jsx

```diff
@@ -41,8 +40,8 @@ function Azimuth(props) {
       {/* TODO: Refactor so "Visualization" is not included... */}
       {/* TODO: ... and then remove the <br> as well. */}
       <br />
-      <VisualizationWrapper vitData={config.vitessce_conf} />
-    </SectionContainer>
+      <VisualizationWrapper vitData={config.vitessce_conf} shouldDisplayHeader={false} />
```

Comment:
```suggestion
      <VisualizationWrapper vitData={config.vitessce_conf} />
```
Better to just leave false booleans off? I thought there were linting warnings about this.

Reply:
I added a default prop `shouldDisplayHeader: true` since all of our other instances display a header.